### PR TITLE
feat(new integration): onboarding sendinblue web device mode destination

### PR DESCRIPTION
**Pull request by Gauravudia**
## PR Description

Onboarding Sendinblue web device mode destination

## Notion ticket

[Ticket link](https://www.notion.so/rudderstacks/SendinBlue-Device-Mode-Destination-f039bdd648c44be5ba857a4c851876b7#25cca8758ad04446b9ada78cb68ea656)

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
Screenshot 2022-12-15 at 1 24 12 PM
Screenshot 2022-12-15 at 1 24 53 PM
Screenshot 2022-12-15 at 1 26 26 PM

- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/804)

### CODE REVIEW

**Review by MoumitaM — APPROVED** [review]

**Review by MoumitaM — APPROVED** [review]

**Review by shrouti1507 — DISMISSED** [review]
Please solve the sonarlint error

**Comment by Gauravudia** [comment]
> Please solve the sonarlint error

`Third party loading code snippet` comment added in sonarcloud.

File: src/integrations/Sendinblue/browser.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      const { phone } = getDefinedTraits(message);
+      if (phone && !validatePhoneWithCountryCode(phone)) {
+        logger.error('[Sendinblue]:: provided phone number is invalid');
+        return;
+      }
+    }
+
+    const userTraits = this.sendTraitsInTrack
+      ? prepareUserTraits(message, this.contactAttributeMapping)
+      : {};
```
Comment (ItsSudip) [inline src/integrations/Sendinblue/browser.js:111]:
```suggestion
    let userTraits = {};
    if (this.sendTraitsInTrack) {
      const { phone } = getDefinedTraits(message);
      if (phone && !validatePhoneWithCountryCode(phone)) {
        logger.error('[Sendinblue]:: provided phone number is invalid');
        return;
      }
      userTraits = this.sendTraitsInTrack
      ? prepareUserTraits(message, this.contactAttributeMapping)
      : {};

    }

```

**Review by ItsSudip — CHANGES_REQUESTED** [review]

File: src/integrations/Sendinblue/browser.js (hunk 1 above)
Comment (Gauravudia) [reply src/integrations/Sendinblue/browser.js]: updated

**Review by ItsSudip — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**